internal/gitserver/gitdomain/exec.go

```diff
@@ -83,7 +84,7 @@ func IsAllowedGitCmd(args []string) bool {
 	allowedArgs, ok := gitCmdAllowlist[cmd]
 	if !ok {
 		// Command not allowed
-		log15.Warn("IsAllowedGitCmd: command not allowed", "cmd", cmd)
+		logger.Warn("IsAllowedGitCmd: command not allowed", log.String("cmd", cmd))
```

Comment:
We have the scope already so the message prefix is no longer needed :)

```suggestion
		logger.Warn("command not allowed", log.String("cmd", cmd))
```